src/server/database-post-register.js

```diff
@@ -7,25 +7,25 @@ const createDatabaseUrl = function() {
   return `${address}:${port}/${databaseName}`;
 };
 
-const postRegister = function(body, callback) {
+const postRegister = function(postUsername, hashPassword, callback) {
   const url = createDatabaseUrl();
   MongoClient.connect(url, function(err, db) {
-    const searchUserName = {username: body.username};
+    const searchUserName = {username: postUsername};
     if (err === null) {
-      let collection = db.collection('heartbeat');
+      let collection = db.collection('register');
       collection.find(searchUserName).toArray(function(err, docs) {
         if (docs.length > 0) {
           callback('409');
         } else {
-          collection.insertOne(body, function(err, docs2) {
+          collection.insertOne({username: postUsername, password: hashPassword}, function(err, docs2) {
```

Comment:
could use `docs` instead of `docs2` because its in a different scope, also will `insertOne` return a bunch of documents or only one?